### PR TITLE
Images and angles

### PR DESCRIPTION
**Pull request by mhelmetag**
Changes:
* Adds basic geometric sprites/spritesheets for player 1 and 2
* Adds new `angle` param
* Uses new angle param to complete classic setup

Screen Shot 2022-05-10 at 10 55 16 PM

### CODE REVIEW

File: src/state/buildBoard.js [hunk 1]
```diff
@@ -26,31 +26,83 @@ export default function buildBoard(gameType = "classic") {
...
       board[0][6] = new PieceBoss({ type: ANUBIS, player: PLAYER_ONE });
-      board[0][7] = new PieceBoss({ type: PYRAMID, player: PLAYER_ONE });
-
-      board[1][2] = new PieceBoss({ type: PYRAMID, player: PLAYER_ONE });
-
-      board[2][3] = new PieceBoss({ type: PYRAMID, player: PLAYER_TWO });
+      board[0][7] = new PieceBoss({
+        type: PYRAMID,
+        player: PLAYER_ONE,
+        angle: 90,
```
Comment (Bates550) [inline src/state/buildBoard.js]: Would it make sense to abstract this to `direction`? Seems like we'd only ever want it to be one of four things and as angle it's not immediately obvious if it's rotated CW or CCW and which direction that ultimately leaves it in.

```ts
      board[0][7] = new PieceBoss({
        type: PYRAMID,
        player: PLAYER_ONE,
        direction: DIRECTIONS.DOWN,
      });
```

**Review by Bates550 — COMMENTED** [review]
Looks great! One comment on `angle`. Nice work on sprites!

File: src/state/buildBoard.js (hunk 1 above)
Comment (mhelmetag) [reply src/state/buildBoard.js]: Ahhh, that's a great point.

I think we have to decide what is up and down (facing which way). For now... maybe depending on what player you are it always orients you as the bottom? Could work that way for multiplayer. I think that's a great idea. It's definitely only multiples of 90 and they should be abstracted for ease of understanding.

I'll make that change.

Comment (mhelmetag) [reply src/state/buildBoard.js]: I did it but... I'm still not sold... what is "up"? How would I know? All the rotational pieces are angled at 45 degrees....

Food for thought.

**Pull request merged**